app/routes/admin/email/components/EmailPage.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+
+import React, { type Node } from 'react';
+import NavigationTab from 'app/components/NavigationTab';
+import NavigationLink from 'app/components/NavigationTab/NavigationLink';
+import Content from 'app/components/Layout/Content';
+import { Link } from 'react-router';
+
+type Props = {
+  children: Node,
+  groups: Array<Object>,
+  location: { pathname: string },
+  params: { groupId: string }
+};
+
+const EmailPage = ({ groups, children, location, params }: Props) => {
+  let createAction;
+  switch (location.pathname) {
+    case '/admin/email/lists':
+      createAction = (
```

Comment:
move this switch to a function so you can use return instead of mutating `createAction`